tbprobe.c

```diff
@@ -107,9 +107,13 @@ unsigned TB_LARGEST = 0;
 #define file(s)                 ((s) & 0x07)
 #define board(s)                ((uint64_t)1 << (s))
 static inline unsigned _lsb(uint64_t b) {
+#if defined(INLINEASM) && !defined(__WIN32__)
   size_t idx;
 __asm__("bsfq %1, %0": "=r"(idx):"rm"(b));
   return idx;
+#else
+	return LSB(b);		 
+#endif
```

Comment:
Not sure why LSB macro/function is used here in the first place. So this maybe wrong.